app/mm-redux/reducers/entities/channels.ts

```diff
@@ -693,4 +712,7 @@ export default combineReducers({
 
     // object where every key is the channel id and has an object with the channel moderations
     channelModerations,
+
+    // object where every key is the channel id containing map of <group_id: ChannelMemberCountByGroup>
```

Comment:
I did not see it, but is this a `new Map()` or are you talking about an `object` here ? those are two different things and in case of a Map you'll have to write a persist transform

Reply:
Nope! Wrong wording. Not map. It's an object.